graphql/pod_resolver.go

```diff
@@ -41,6 +43,28 @@ func (r *podResolver) Task(ctx context.Context, obj *model.APIPod) (*model.APITa
 	return apiTask, nil
 }
 
+func (r *podResolver) Events(ctx context.Context, obj *model.APIPod, limit *int, page *int) (*PodEvents, error) {
+	events, count, err := event.MostRecentPaginatedPodEvents(utility.FromStringPtr(obj.ID), *limit, *page)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("fetching pod events: %s", err.Error()))
+	}
+	// populate eventlogs pointer arrays
```

Comment:
Do we need this comment? It seems somewhat self-evident that the for loop is building a slice.

Reply:
Seems like if this comment doesn't add anything, we should remove it.